### ISSUE TITLE
The service discovery page has a malformed table 

### ISSUE THREAD

**Issue by bryanhuhta** [pyroscope-io/pyroscope#1624]
## What

When I browse to the service discovery page, the table is malformed whenever the Kube pod has long labels. This makes it quite difficult to read some of the values (like "Scrape URL" or "Last error"). As a workaround, I will either use the element inspector or copy-paste the section in question to a text editor. Here is a screenshot of this:

![Screen Shot 2022-10-17 at 1 31 39 PM](https://user-images.githubusercontent.com/32787160/196287134-6d9828e1-b049-4365-a0d0-c1fe81396425.png)

> Note: I redacted a lot of the label keys and values by request of our security team, so apologies for the blurring.

## Possible improvement

It might be possible to solve this by giving each badge a fixed width and using `text-overflow: ellipsis`:

```css
.badge {
  max-width: 10rem;
  text-overflow: ellipsis;
  overflow: hidden;

  /* ... other properties ... */
}
```

Which would produce a result similar to this:

![Screen Shot 2022-10-17 at 1 34 03 PM](https://user-images.githubusercontent.com/32787160/196289318-5940134b-c909-48f1-ac75-848663b7b3ed.png)

Though, this would probably require a hover preview for each tag since many tags won't show their entire contents. This alone may be too much of a bad UX to want to pursue this approach.

## Pyroscope Version

v0.31

## Chrome Version

Version 106.0.5249.119 (Official Build) (x86_64)

**Comment by Rperry2174** [comment]
Thanks for reporting @bryanhuhta and thanks for the suggestion we'll take a look and have this fixed in the next release!

**Comment by Rperry2174** [comment]
Here's what I think we should try I think we could assume a screen size like 1600x900 (debatable):

- allocate a column minimum width thats reasonable for all the "short" columns (i.e. health, scrape duration, last error don't need too much space. Their headers could also be on two lines to allow for even smaller width)
- allocate a column minimum width that has a little extra space for the "medium" column (scrape url)
- allocate a column minimum width the rest of the space for the most important columns (discovered labels / labels) 

Once we figure out what those values are I think we should make the max-width of the "pills" to be slightly smaller than the width of the columns they're in and then either:
1. overflow vertically so the pills end up being multiple lines if they're really long
2. make the pills have ellipses and then on-hover make them overflow to multiple lines 

wdyt @pavelpashkovsky ? btw to test this I ran with [mattermost example](https://github.com/pyroscope-io/pyroscope/blob/main/examples/golang-pull/mattermost/README.md) and just edited values to make them long

**Comment by Rperry2174** [comment]
fixed in #1626 let us know if you have any other feedback @bryanhuhta ! This will be included in next release
